jekyll/_cci2/server-3-faq.adoc

```diff
@@ -0,0 +1,118 @@
+---
+version:
+- Server v3.x
+- Server Admin
+---
+= CircleCI Server v3.x Operations Overview
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+The following guide contains information useful for CircleCI Server Operators.
+
+It is assumed that you have already read the xref:server-3-overview.adoc[Server 3.x Overview].
+
+CircleCI Server schedules CI jobs using the https://www.nomadproject.io/[Nomad] scheduler. The Nomad control plane runs inside of Kubernetes, while the
+Nomad clients are provisioned outside the cluster. The Nomad clients need access to the Nomad control plane, output processor,
+and VM service.
+
+CircleCI Server can run Docker jobs on the Nomad clients, but it can also run jobs in a dedicated VM. These VM jobs are
+controlled by the Nomad clients, therefore the Nomad clients must be able to access the VM machines on port 22 for SSH
+and port 2376 for remote Docker jobs.
+
+Job artifacts and output are sent directly from jobs in Nomad to object storage (S3, GCS, or other supported options).
+Audit logs and other items from the application are also stored in object storage so both the Kubernetes cluster and the
+Nomad clients will need access to object storage.
+
+toc::[]
+
+## Build Environment
+
+CircleCI Server 3.x uses Nomad as the primary job scheduler. Refer to our xref:nomad.adoc[Introduction to Nomad Cluster Operation]
+to learn more about the job scheduler and how to perform basic client and cluster operations.
+
+By default, CircleCI Nomad clients automatically provision compute resources according to the executors configured for
+each job in a project’s `.circleci/config.yml` file.
+
+### Nomad Clients
+
+Nomad Clients run without storing state, enabling you to increase or decrease the number of containers as needed.
+
+To ensure enough Nomad clients are running to handle all builds, track the queued builds and increase the number of
+Nomad Client machines as needed to balance the load. For more on tracking metrics see Monitoring Your Installation.
+
+Each machine reserves two vCPUs and 4GB of memory for coordinating builds. The remaining processors and memory create the
+containers for running jobs. Larger machines are able to run more containers and are limited by the number of available
+cores after two are reserved for coordination.
+
+NOTE: The maximum machine size for a Nomad client is 128GB RAM/64 CPUs. Contact your CircleCI account representative to
+request use of larger machines for Nomad Clients.
+
+The following table describes the ports used on Nomad clients:
+
+--
+[.table.table-striped]
+[cols=3*, options="header", stripes=even]
+|===
+| Source
+| Ports
+| Use
+
+| End Users
+| 64535-65535
+| SSH into builds
+
+| Administrators
+| 80 or 443
+| CCI API Access
+
+| Administrators
+| 22
+| SSH
+
+| Services Machine
+| all traffic, all ports
+| Internal communications
+
+| Nomad Clients (including itself)
```

Comment:
The nomad clients don't talk to each other so I think this can be removed. I believe we talk about all the port requirements for nomad in the server documentation. Perhaps best to copy what we have there?